### PR TITLE
faq: added an entry about phantom monitor

### PR DESCRIPTION
**Pull request by primalmotion**
add an entry in the FAQ about phantom monitor and a potential workaround

### CODE REVIEW

**Comment by vaxerski** [comment]
I'd also mention "disappearing" workspaces

**Comment by primalmotion** [comment]
done

File: pages/FAQ/_index.md [hunk 1]
```diff
@@ -383,3 +383,19 @@ from Steam.
 
 Steam has its own file picker, however, it's not functional. Install
 `xdg-desktop-portal-gtk` to show the desktop's file picker.
+
+# Workspaces or clients are disappearing or monitor related dispatchers cause crashes
+
+It seems there is a Kernel bug making the system thinks there is an extra
```
Comment (vaxerski) [inline pages/FAQ/_index.md:389]: think

File: pages/FAQ/_index.md [hunk 2]
```diff
@@ -383,3 +383,19 @@ from Steam.
...
+monitor.
+
+First check the list of monitors detected by Hyprland by running:
+
+`hyprctl monitors`
+
+If you see a monitor that should not be there (usually named `Unknown-1`), you
+can work around the issue by adding in your `hyprland.conf`:
+
+`monitor=Unknown-1,disabled`
```
Comment (vaxerski) [inline pages/FAQ/_index.md:401]: make it a full line code block with ini please
```ini
like=this
```

Comment (primalmotion) [reply pages/FAQ/_index.md]: done

**Review by vaxerski — APPROVED** [review]
tanks!

**Pull request merged**